src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1401,12 +1401,6 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
-            JToken hostHealthMonitorEnabled = (JToken)config["hostHealthMonitorEnabled"];
-            if (hostHealthMonitorEnabled != null && hostHealthMonitorEnabled.Type == JTokenType.Boolean)
-            {
-                scriptConfig.HostHealthMonitorEnabled = (bool)hostHealthMonitorEnabled;
```

Comment:
Removing this old setting in favor of a config section block for host monitor. Since the feature is currently completely disabled in production, removing this setting in favor of the new config format won't be an issue.